ci-operator/step-registry/gather/installer/gather-installer.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+
+echo "Get the installer gather logs if it is around"
+
+dir=/tmp/installer
```

Comment:
I am not sure that dir exists in your step container

Reply:
Updated container to: installer